pkg/hostagent/pods.go

```diff
@@ -443,6 +446,40 @@ func (agent *HostAgent) podChanged(podkey *string) {
 	agent.podChangedLocked(podobj)
 }
 
+func addPolicy(gset map[metadata.OpflexGroup]bool, g []metadata.OpflexGroup,
```

Comment:
seems this code is repeated similar to addGroup.  you can move both these functions to group_assign.go

Reply:
done